sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Grammar - join last two sentences, reword slightly: <code>benchmarking the endpoint. We also highly recommend you to specific</code> -&gt; <code>benchmarking the endpoint, we also highly recommend that you specify</code></p><p> </p><p>Elaborate a little on the consequences of not providing these parameters. What happens when the recommendation is worse? Slower inference, and/or something else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Fixed.